### ISSUE TITLE
respiro openapi

### ISSUE THREAD

**Issue by yoyo2k** [feketemihai/l10n-romania#42]
cum zicea @feketemihai  in pull #37 

```
def check_vat_subjected(self):
    if self.vat_on_payment is True:
        self.vat_subjected = True
    else:
        self.vat_subjected = self._check_vat_subjected()
```

ar fi bine asa ca solutie?

**Comment by feketemihai** [comment]
Da...din partea mea e ok...

**Comment by feketemihai** [comment]
Fixed in #45
